### PR TITLE
fix: optimize feedback and app icon

### PR DESCRIPTION
**Pull request by jiaowoxiaobala**
1. optimize feedback
img

2. app icon
img

3. styles
img
img

### CODE REVIEW

File: web/i18n/lang/common.en.ts [hunk 1]
```diff
@@ -44,6 +44,8 @@ const translation = {
     copySuccessfully: 'Copied successfully',
     paySucceeded: 'Payment succeeded',
     payCancelled: 'Payment cancelled',
+    generatedSuccessfully: 'Generated Successfully',
+    generationFailed: 'Generation Failed',
```
Comment (crazywoola) [inline web/i18n/lang/common.en.ts]: Keep one of them Generation or Generated

File: web/app/(commonLayout)/app/(appDetailLayout)/[appId]/overview/cardView.tsx [hunk 2]
```diff
@@ -88,7 +91,11 @@ const CardView: FC<ICardViewProps> = ({ appId }) => {
         url: `/apps/${appId}/site/access-token-reset`,
       }) as Promise<UpdateAppSiteCodeResponse>,
     )
-    handleError(err)
+
+    handleError(err, {
```
Comment (crazywoola) [inline web/app/(commonLayout)/app/(appDetailLayout)/[appId]/overview/cardView.tsx]: Keep one of them Generation or Generated

File: web/app/(commonLayout)/app/(appDetailLayout)/[appId]/overview/cardView.tsx [hunk 3]
```diff
@@ -33,18 +33,21 @@ const CardView: FC<ICardViewProps> = ({ appId }) => {
   if (!response)
     return <Loading />
 
-  const handleError = (err: Error | null) => {
+  const handleError = (err: Error | null, msgs = {
+    success: 'modifiedSuccessfully',
```
Comment (crazywoola) [inline web/app/(commonLayout)/app/(appDetailLayout)/[appId]/overview/cardView.tsx]: Keep one of them Modified or Modification

**Review by crazywoola — CHANGES_REQUESTED** [review]

File: web/app/(commonLayout)/app/(appDetailLayout)/[appId]/overview/cardView.tsx (hunk 3 above)
Comment (jiaowoxiaobala) [reply web/app/(commonLayout)/app/(appDetailLayout)/[appId]/overview/cardView.tsx]: ok

File: web/app/(commonLayout)/app/(appDetailLayout)/[appId]/overview/cardView.tsx [hunk 4]
```diff
@@ -33,18 +33,21 @@ const CardView: FC<ICardViewProps> = ({ appId }) => {
...
+    fail: 'modificationFailed',
```
Comment (MatriQ) [inline web/app/(commonLayout)/app/(appDetailLayout)/[appId]/overview/cardView.tsx]: `success` and `filed` should not appear at the same time,  so I think defining this function like the suggestion below would make more sense.
```typescript
const handleError = (param: 
    | { success: 'modifiedSuccessfully',  err: null }
    | { failed: 'modificationFailed', err: Error
  }) => { *** }
```

Comment (jiaowoxiaobala) [reply web/app/(commonLayout)/app/(appDetailLayout)/[appId]/overview/cardView.tsx]: > `success` and `filed` should not appear at the same time, so I think defining this function like the suggestion below would make more sense.
> 
> ```ts
> const handleError = (param: 
>     | { success: 'modifiedSuccessfully',  err: null }
>     | { failed: 'modificationFailed', err: Error
>   }) => { *** }
> ```

@MatriQ  you are right ！！

**Comment by jiaowoxiaobala** [comment]
@crazywoola ok，plz review again

**Review by crazywoola — APPROVED** [review]

**Pull request merged**